ui/tests/acceptance/policies/index-test.js

```diff
@@ -120,8 +121,10 @@ module('Acceptance | policies/acl', function (hooks) {
       'navigates to policy show on successful save'
     );
     assert.dom(SELECT.policyTitle).hasText(policyLower, 'displays the policy name on the show page');
-    // will fail if you have a license about to expire.
```

Comment:
It wasn't clear to me how this comment related to whether or not a flash message showed. Acceptance tests tend to rack up the flash messages, so opted to spy + assert for a flash message we _do_ expect
<img width="501" alt="Screenshot 2024-06-10 at 6 47 42 PM" src="https://github.com/hashicorp/vault/assets/68122737/25f0b305-f286-4886-ae15-d37ce6d8d0cb">